Code/Framework/AzCore/AzCore/Asset/AssetCommon.h

```diff
@@ -82,6 +83,10 @@ namespace AZ
             static AssetId CreateString(AZStd::string_view input);
             static void Reflect(ReflectContext* context);
 
+            static constexpr size_t MaxStringBuffer = AZ::Uuid::MaxStringBuffer + 9; /// UUid size (includes terminal) + ":" + hex, subId
```

Comment:
extreme nitpick: UUid -> Uuid
```suggestion
            static constexpr size_t MaxStringBuffer = AZ::Uuid::MaxStringBuffer + 9; /// Uuid size (includes terminal) + ":" + hex, subId
```